# Inferential statistics

In [ ]:
It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?
# 1. Set up the Hypotheses:

# In this scenario, you are tasked with performing a hypothesis test to determine whether the group in the Honolulu Heart Study has a significantly different systolic blood pressure from the regular population, whose mean systolic blood pressure is assumed to be 120 mm Hg.


In [1]:
import math

# Given values
mu = 120  # Population mean
x_bar = 130.1  # Sample mean
sigma = 21.21  # Sample standard deviation
n = 100  # Sample size

# Calculate the Z-test statistic
Z = (x_bar - mu) / (sigma / math.sqrt(n))

# Print the result
print("Z-test statistic:", Z)


Z-test statistic: 4.761904761904759


# 6. Make a Decision:

Now that we have calculated the Z-value to be approximately 4.76, we compare this with the critical
Z-value for a two-tailed test at 𝛼 = 0.05

For a two-tailed test with 𝛼 = 0.05, the critical Z-value is approximately ±1.96. If the calculated
Z-value exceeds ±1.96, we reject the null hypothesis.

Since 4.76 is much greater than 1.96, we reject the null hypothesis.

Conclusion:

There is significant evidence to suggest that the systolic blood pressure in the sample from the 
Honolulu Heart Study is significantly different from the population mean of 120 mm Hg.

